cmd/app.go

```diff
@@ -658,40 +673,45 @@ func templateInstall(ctx *cli.Context) error {
 		app.TargetNamespace = namespace
 	}
 
+	app.Wait = ctx.Bool("wait")
+	// 300 seconds is the default value that helm already uses
+	if timeout := ctx.Int64("timeout"); timeout != 300 {
+		app.Timeout = timeout
+	}
+
 	madeApp, err := c.ProjectClient.App.Create(app)
 	if err != nil {
 		return err
 	}
 
-	// wait for the app to be installed to print the notes
-	var (
-		timeout   int
-		installed bool
-	)
-	for !installed {
-		if timeout == 60 {
-			return errors.New("timed out waiting for app to install. Run 'rancher apps' to verify status")
-		}
-		timeout++
-		time.Sleep(2 * time.Second)
-		madeApp, err = c.ProjectClient.App.ByID(madeApp.ID)
-		if err != nil {
-			return err
-		}
-		for _, condition := range madeApp.Conditions {
-			condType := strings.ToLower(condition.Type)
-			condStatus := strings.ToLower(condition.Status)
-			if condType == "installed" && condStatus == "true" {
-				installed = true
-				break
-			}
-		}
+	fmt.Printf("run \"app show-notes %s\" to view app notes once app is ready\n", madeApp.Name)
+
+	return nil
+}
+
+// appNotes prints notes from app's notes.txt file
+func appNotes(ctx *cli.Context) error {
+	c, err := GetClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	if ctx.NArg() < 1 {
+		return cli.ShowSubcommandHelp(ctx)
 	}
 
-	if len(madeApp.Notes) != 0 {
-		fmt.Println(madeApp.Notes)
+	resource, err := Lookup(c, ctx.Args().First(), "app")
+	if err != nil {
+		return err
 	}
 
+	app, err := c.ProjectClient.App.ByID(resource.ID)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println(app.Notes)
```

Comment:
do a len check like before and print no notes or something if they don't exist. Printing a blank line will confuse people.